src/validation/userValidation.js

```diff
@@ -0,0 +1,21 @@
+import { check } from 'express-validator';
+import { loginErrors } from '../utils/errorMessage';
+import Errors from './errorMessage';
+
+const loginValidation = [
+  check('email')
+    .exists({ checkFalsy: true })
+    .withMessage(`email: ${loginErrors.undefinedEmail}`)
+    .isEmail()
+    .withMessage(`email: ${loginErrors.invalidEmail}`)
+    .matches(/@andela.com$/)
+    .withMessage(`email: ${loginErrors.nonAndelanEmail}`),
+  check('password')
+    .exists({ checkFalsy: true })
+    .withMessage(`password: ${loginErrors.undefinedPassword}`)
+    .isLength({ min: 8 })
```

Comment:
I don't think we need to validate if the user's password is up to 8 characters when logging in. Please remove this.

Reply:
ok will remove it now

Reply:
thanks for the feedback, will follow the instructions